lucetc/src/lib.rs

```diff
@@ -31,6 +29,7 @@ pub use crate::{
     patch::patch_module,
 };
 use failure::{format_err, Error, ResultExt};
+use lucet_module_data::bindings::Bindings;
```

Comment:
since Bindings are basically a requirement to compile a module, I would `pub use` this so that most clients can just depend on `lucetc`